qemu/tests/cfg/virtio_fs_share_data_hugepage.cfg

```diff
@@ -65,11 +65,20 @@
         - with_cache:
             variants:
                 - auto:
-                    fs_binary_extra_options += ",cache=auto"
+                    !Host_RHEL.m8:
+                        fs_binary_extra_options += " --cache=auto"
+                    Host_RHEL.m8:
+                        fs_binary_extra_options += " -o cache=auto"
                 - always:
-                    fs_binary_extra_options += ",cache=always"
-                - none:
-                    fs_binary_extra_options += ",cache=none"
+                    !Host_RHEL.m8:
+                        fs_binary_extra_options += " --cache=always"
+                    Host_RHEL.m8:
+                        fs_binary_extra_options += " -o cache=always"
+                - never:
+                    !Host_RHEL.m8:
+                        fs_binary_extra_options += " --cache=never"
+                    Host_RHEL.m8:
+                        fs_binary_extra_options += " -o cache=none"
```

Comment:
Hi @zhaojunyao7 @hellohellenmao @xiagao 
Does it make sense that we set the `fs_binary_extra_options += " -o cache=none"` for RHEL 8 host mandatory?
And from the @maxujun 's comment, just there is no such usage `-o cache=<auto, always, none>` on PPC, so do we need to extend this limitation to all the platforms? or this usage does work with other platforms neither.
Thanks.

Reply:
Yes, for all the platform.